tools/scripts/ansible/templates/deploy-sh.j2

```diff
@@ -4,9 +4,11 @@
 if [[ ${1} = "--repo" ]];
 then
     shift
-    GIT_PREFETCH=https://github.com/$(echo -n "${1}" | tr -c 'A-Za-z0-9_.-/' '-').git
+    GITHUB_REPO=$(echo -n "${1}" | tr -c 'A-Za-z0-9_.-/' '-')
+    GIT_PREFETCH=https://github.com/${GITHUB_REPO}.git
     shift
 else
+    GITHUB_REPO=unicode-org/cldr
```

Comment:
ideally avoid "github" in name, in case we switch to different provider -- e.g., SOURCE_REPO

Reply:
Except that this is actually a github repo, and this is called from github. If we changed we would need to redo this script. The parameter comes in as USER/REPO from github.